ouroboros-network/src/Ouroboros/Network/PeerSelection/LedgerPeers.hs

```diff
@@ -305,26 +302,36 @@ ledgerPeersThread inRng dnsSemaphore toPeerAddr tracer readUseLedgerAfter
         let age = diffTime now oldTs
         (peerMap', bigPeerMap', ts) <- if age > peerListLifeTime
                              then
-                                 case useLedgerAfter of
-                                   DontUseLedger -> do
+                                 case useLedgerPeers of
+                                   DontUseLedgerPeers -> do
                                      traceWith tracer DisabledLedgerPeers
                                      return (Map.empty, Map.empty, now)
-                                   UseLedgerAfter slot -> do
+                                   UseLedgerPeers Always -> do
+                                     peers <- (\case
+                                                BeforeSlot          -> []
+                                                LedgerPeers _ peers -> peers
+                                              )
+                                          <$> atomically (getLedgerPeers ledgerPeersConsensusInterface 1)
```

Comment:
Alternatively we could ask the consensus team to redefine [this function](https://github.com/input-output-hk/ouroboros-consensus/blob/c9af2a29e800771329516375e6d0569cac21c594/ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/NodeKernel.hs#L661).  This would be more logical because `Slot 0` does not exit.